extide/gradle/netbeans-gradle-tooling/src/main/java/org/netbeans/modules/gradle/tooling/NbProjectInfoBuilder.java

```diff
@@ -308,6 +307,7 @@ private String getTaskInheritance(Task t) {
         "shouldRunAfter",
         "enabled",
         "description",
+        "vendor",
```

Comment:
just as note: this fixes the cycle (OOME) by excluding the `updateDaemonJvm.vendor.*` path

this had similar symptoms as https://github.com/apache/netbeans/issues/6674#issuecomment-1803932384 but surfaced as OOME instead of stack overflow.

Reply:
Let me propose a different fix -- I've debugged this and (mea culpa !!!) it's caused by serializing STATIC properties reported by the metaobject. And in the `JvmVendorSpec` case, each instance has another set of 13 properties. Although _recursive_ serialization of the same instance is guarded ... combinatiorial explosion is not.